pyproject.toml

```diff
@@ -45,7 +47,7 @@ line_length = 79
 [tool.flake8]
 max-complexity = 15
 max-line-length = 79
-extend-ignore = "E203"
+extend-ignore = ["E203", "E731"]
```

Comment:
Please do not ignore E731 in general but ignore in a specific case if necessary.

Reply:
Checkout this explanation as a reference https://www.flake8rules.com/rules/E713.html

Reply:
I think it is this one: https://www.flake8rules.com/rules/E731.html

Reply:
The preferred way would be to change the location which fails because of the rule. If it's not possible to go by the rule you can append `# noqa` to the line to make flake8 ignore it.

Reply:
Changed